styles/octocat.css

```diff
@@ -11,3 +11,73 @@ ul {
 * {
   box-sizing: border-box;
 }
+
+img {
+  max-width: 100%;
+}
+
+strong {
+  color: black;
+}
+
+.prompt {
+  text-align: center;
+}
+
+.section {
+  padding: 7rem;
+}
+
+.container {
+  max-width: 65rem;
+  margin: 0 auto;
+}
+
+.row {
+  margin: -2rem;
```

Comment:
Match the negative margin of your row to the `margin` value of the `.grid-item` within it to make sure that this will line up to other content within containers.